core/validation/failures/user.py

```diff
@@ -75,7 +76,7 @@ def __str__(self):
 class WrongTypeFailure(SchemaUserValidationFailure):
     """Class representing a wrong type failure that is raised when data is of an incorrect type."""
 
-    expected_type: str
+    expected_type: str | float | int | datetime | object | bool | list
```

Comment:
For now this doesn't include list. Only ```Postcodes``` is a list, and it's a column populated by our own extraction function - and converted into a list from a series of extracted strings of postcodes - so we don't want to validate that the type is correct and return a message to the user about this.